hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2347,7 +2347,7 @@ public OmKeyInfo lookupKey(OmKeyArgs args) throws IOException {
     boolean auditSuccess = true;
     try {
       metrics.incNumKeyLookups();
-      return keyManager.lookupKey(args);
+      return keyManager.lookupKey(args, getClientAddress());
```

Comment:
Do we assume the topology mapping always use Ip address?

Reply:
No, use Ip address or hostname depends on the value of   "dfs.datanode.use.datanode.hostname" property. If it's true, then use hostname, if it's false, the use Ipaddress. The default value of this property is false.